e2e/test-environment/__tests__/esmDefault.test.js

```diff
@@ -5,9 +5,14 @@
  * LICENSE file in the root directory of this source tree.
  *
  * @jest-environment ./EsmDefaultEnvironment.js
+ * @my-custom-pragma pragma-value
  */
 'use strict';
 
 test('access env', () => {
   expect(property).toBe('value'); // eslint-disable-line no-undef
 });
+
+test('docblock pragmas', () => {
```

Comment:
Would prefer a distinct test file and Environment impl to separate concerns - this one tests ESM default export support and I wouldn't want `docblock pragmas` to fail as well if that breaks

Reply:
Done.